src/main/java/org/hobbit/utils/kubernetes/KubeHelper.java

```diff
@@ -0,0 +1,40 @@
+package org.hobbit.utils.kubernetes;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+public class KubeHelper {
+    private static final Logger LOGGER = LoggerFactory.getLogger(KubeHelper.class);
+    /**
+     * Gets dns friendly ip
+     *
+     * @return plain hostname (or address)
+     */
+    public static String getDnsFriendlyIP(String namespace) {
+        if(namespace == null) {
+            namespace = "default";
+        }
+        String ip =  getPodIP();
+        if(!ip.equals("unknown")&&ip!=null) {
+            String dnsFriendlyIp = ip.replace(".","-")+namespace+".pod.cluster.local";
+            return dnsFriendlyIp;
+        }else {
+            LOGGER.error("Error getting pod IP: {} namespace: {}", ip, namespace);
+            return "unknown";
+        }
+    }
+
+    public static String getPodIP() {
```

Comment:
`getContainerIp`

Reply:
because we define another class (DNSHelper ) then here we just talk about Kubernetes , then  Pod is a better match